Assets/GameBrowserEntry.cs

```diff
@@ -53,20 +54,8 @@ public void OnClick()
         MainHandler.Instance.SetScene(MainHandler.SceneType.MainScene);
     }
 
-    IEnumerator GetTexture(Image target, string url)
+    public async void GetImage(string url, Image target)
```

Comment:
This is a generic method that could be used in other contexts outside the GameBrowserEntry class. It should also be marked as static as it doesn't rely on an instance. Could possibly be in the FumbblApi class since that's the only source we use for images.